src/reanimated2/animation/transformationMatrix/matrixUtils.tsx

```diff
@@ -34,6 +34,7 @@ export function isAffineMatrixFlat(x: unknown): x is AffineMatrixFlat {
   );
 }
 
+// ts-prune-ignore-next This function is exported to be tested
```

Comment:
This should be moved to a separate PR

Reply:
🆗 So here: https://github.com/software-mansion/react-native-reanimated/pull/4839